src/main/java/reposense/model/CliArguments.java

```diff
@@ -13,6 +13,7 @@
     protected Optional<Date> sinceDate;
     protected Optional<Date> untilDate;
     protected List<String> formats;
+    protected boolean isAutomaticallyLaunching;
```

Comment:
Please add this to `equals()`.

`this.isAutomaticallyLaunching == (otherCliArguments.isAutomaticallyLaunching)`

Reply:
thanks for the catch, made the change.